backend/api/MQTT/MessageModels/IsarTelemetryUpdate.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Text.Json.Serialization;
+using Api.Services.Models;
+
+namespace Api.Mqtt.MessageModels
+{
+#nullable disable
+    public class IsarTelemetyUpdateMessage : MqttMessage
```

Comment:
Rename to something making it clear that this is the isar message updating the media config